homeassistant/components/alexa/handlers.py

```diff
@@ -1523,3 +1524,30 @@ def temperature_from_object(hass, temp_obj, interval=False):
     )
 
     return directive.response()
+
+
+@HANDLERS.register(("Alexa.CameraStreamController", "InitializeCameraStreams"))
+async def async_api_initialize_camera_stream(hass, config, directive, context):
+    """Process a InitializeCameraStreams request."""
+    entity = directive.entity
+    camera_component = hass.data.get(camera.DOMAIN)
+    camera_entity = camera_component.get_entity(entity.entity_id)
+    stream_source = await camera.async_request_stream(
+        hass, camera_entity.entity_id, fmt="hls"
+    )
+    payload = {
+        "cameraStreams": [
+            {
+                "uri": f"{config.camera_stream_url}{stream_source}",
```

Comment:
Let's use `helpers.async_get_external_url(hass)` for this.

Reply:
For manual set up, we should ask people to set up their base url.

Reply:
Added `helpers.async_get_external_url(hass)` in the `alexa.AlexaConfig` class if this value isn't set. I still need to add it to the `cloud.AlexaConfig` class.